bulk-ingester/src/main/java/org/alfresco/hxi_connector/bulk_ingester/processor/BulkIngesterConfig.java

```diff
@@ -23,18 +23,23 @@
  * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
  * #L%
  */
-package org.alfresco.hxi_connector.live_ingester;
 
-import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
+package org.alfresco.hxi_connector.bulk_ingester.processor;
 
-@SpringBootApplication
-@SuppressWarnings("PMD.UseUtilityClass")
-public class BulkIngesterApplication
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.PositiveOrZero;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+@Data
+@Validated
+@ConfigurationProperties("alfresco.bulk.ingest.node-params")
+public class BulkIngesterConfig
 {
+    @PositiveOrZero
+    private long fromId;
 
-    public static void main(String[] args)
-    {
-        SpringApplication.run(BulkIngesterApplication.class, args);
-    }
+    @Positive private long toId;
```

Comment:
I'm surprised the formatter doesn't have a preference over inline annotations or multiline. FWIW I prefer annotations on their own line.

Reply:
I think formatter did it :/ I'll try to move it to separate line but idk if it'll work

Reply:
That would be a plot twist if the formatter likes a mixture of same line/separate line :yum:

Reply:
yep it is formatter thing :/